### DISCUSSION TITLE
Command rotate page

### DISCUSSION THREAD

**Discussion by arunki** [sumatrapdfreader/sumatrapdf#2825] [Documentation]
Hi!

There is a command to rotate the page left and right in Sumatra, ctrl+shift+ m or k, also there are buttons for that purpose.

However, when i hit ']', the page rotates to right.

I'd like to disable this key function in my Sumatra PDF. Possible?

Anyone gives me a hand?

**Comment by GitHubRulesOK** [comment]
in latest official and pre-release version you can set custom key `]` to CmdNone (that is Do nothing) see https://www.sumatrapdfreader.org/docs/Customizing-keyboard-shortcuts ,just be sure there is no confusion over the outer pairs of [ and ]

**Reply by arunki** [reply]
Hi!! Thanks so much.

I am inserting these lines:

DisablingKey [
CmdNone ]
]

It does not function yet. Even if i open Sumatra as administrator.

**Reply by GitHubRulesOK** [reply]
it should look like 
```
Shortcuts [
	[
		Cmd = CmdNone
		Key = ]
	]
]
```

**Reply by arunki** [reply]
YEZ!!
Thank you!!
